docs/guides/use_checkpointing.ipynb

```diff
@@ -5,17 +5,25 @@
    "id": "6e9134fa",
```

Comment:
<p><code>flax.training,checkpoints</code> -&gt; <code>flax.training.checkpoints</code> (comma to period)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/2688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>